### ISSUE TITLE
Problem with mashup

### ISSUE THREAD

**Issue by guventum** [balexbyrd/sense-Funnel#2]
Hi, 

There seems to be problem when using this extension in a mashup. 

It gives the following error: 
"Visualization not found: d3Funnel"

Perhaps this has something to do with the script. It may only be used in a Qlik Sense app? 

Can you please help me find a solution for this problem. There ain't any other funnel chart available at this moment. 

Kind Regards, 

Guven Tum

**Comment by balexbyrd** [comment]
Guven,

I've experienced what you're seeing. A work around is in your mashup qlik js file, add d3 as a path, an example is below. It should be located at the top of your script.

```
require.config( {
     
     baseUrl: (config.isSecure ? "https://" : "http://" ) + config.host + (config.port ? ":" + config.port : "" ) + config.prefix + "resources",

     paths: {d3: "//cdnjs.cloudflare.com/ajax/libs/d3/3.5.17/d3"}
});
```

**Comment by guventum** [comment]
Hi Alex,

Thank you for your reply!

Is it possible if I modified the extension by using your code?

[cid:image001.jpg@01D27D6C.2E831C10]


I tried it like the above example, however now instead of an error it only showed me the title:

[cid:image002.jpg@01D27D6C.2E831C10]


Kind regards,

Guven

Met vriendelijke groet, Kind regards,
Güven Tüm
TraineePinkElephant
T. +31882356655
E. guven.tum@pinkelephant.nl
W. www.pinkelephant.nl
Naarden | 's Hertogenbosch | Breda
Op weg naar een doelgerichte IT-Strategie
De inhoud van dit bericht is alleen bestemd voor de geadresseerde, kan vertrouwelijke
of persoonlijke informatie bevatten en is uitdrukkelijk verstuurd onder de voorwaarden
van de volgende e-mail disclaimer
e-mail disclaimer
Van: Alex Byrd [mailto:notifications@github.com]
Verzonden: donderdag 2 februari 2017 15:23
Aan: balexbyrd/sense-Funnel <sense-Funnel@noreply.github.com>
CC: Güven Tüm <guven.tum@pinkelephant.nl>; Author <author@noreply.github.com>
Onderwerp: Re: [balexbyrd/sense-Funnel] Problem with mashup (#2)


Guven,

I've experienced what you're seeing. A work around is in your mashup qlik js file, add d3 as a path, an example is below. It should be located at the top of your script.

require.config( {



     baseUrl: (config.isSecure ? "https://" : "http://" ) + config.host + (config.port ? ":" + config.port : "" ) + config.prefix + "resources",



     paths: {d3: "//cdnjs.cloudflare.com/ajax/libs/d3/3.5.17/d3"}

});
